include/exadg/operators/inverse_mass_operator.h

```diff
@@ -91,9 +109,36 @@ class InverseMassOperator
     }
   }
 
+  void
+  initialize_inverse_mass_operator_with_block_jacobi()
+  {
+    // initialize mass operator
+    dealii::AffineConstraints<Number> const & constraint =
+      matrix_free->get_affine_constraints(dof_index);
```

Comment:
I am not sure whether we want constraints here, see the mass operator in the `Structure` module https://github.com/exadg/exadg/blob/41f8752a19a667b64009fcd6f4325a087afe8baa/include/exadg/structure/spatial_discretization/operator.cpp#L156-L158 for comparison.